envs/hmftools/hmftools-sage-2.6.yaml

```diff
@@ -124,4 +124,4 @@ dependencies:
   - xz=5.2.5
   - zlib=1.2.11
   - zstd=1.4.5
-prefix: /home/kdreval/miniconda3/envs/sage
\ No newline at end of file
+prefix: /home/kdreval/miniconda3/envs/sage
```

Comment:
Should conda environments have this in them?

Reply:
I did this to remove the red circle .

Reply:
IIRC, the red circle at the end of a file in GitHub refers to the lack of a trailing new line.